.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Thank you for creating a pull request!
+
+Please double-check the following.
+
+- Read [our contribution guide](https://github.com/pfnet/optuna/blob/master/CONTRIBUTING.md).
+- Did you write sufficient test code?
+- Did you write sufficient documentation?
```

Comment:
Ah this is the template from Chainer. I think it could be simplified actually, "documentation" already seem covered by the contribution guide, and the "test code" could be too (maybe in a separate PR if that's easier). Then we could simply this a lost just referring the reader to the contribution guide? Something along

```markdown
Thank you for creating a pull request!

Please go through [our contribution guide](https://github.com/pfnet/optuna/blob/master/CONTRIBUTING.md) to hopefully get your changes merged quicker.
```

Reply:
Yes, it's almost from Chainer's template.
Your suggestion makes sense. I'll update `PULL_REQUEST_TEMPLATE.md` and `CONTRIBUTING.md`.

Reply:
Looks great!